### DISCUSSION TITLE
Patching preinstalled YouTube app with LSPatch/Shizuku/ADB or any other without root method

### DISCUSSION THREAD

**Discussion by akashdh11** [ReVanced/revanced-discussions#394] [Question] [answered]
Are there any ways to patch OEM preinstalled YouTube app using
1. LSPatch:  https://github.com/LSPosed/LSPatch
2. Shizuku:  https://shizuku.rikka.app/
3. like can we patch the apk in ROOTED mobile and patch & install that apk using LSPatch  in the NONROOTED phone without actual keystore
4. or any other WITHOUT ROOT method and without installing additional package name changed apk

**Comment by oSumAtrIX** [comment, marked as answer]
ReVanced changes the package name for non root and mounts on top for root.

**Reply by akashdh11** [reply]
Yeah thanks I got your point 🙂. But I am asking is there any way revanced to mount on top of of original YouTube on non rooted phones. By using LSPatch and Shizuku as both work with shell privilege without requiring root

**Reply by TheJeterLP** [reply]
We currently do not support that.
